test/jdk/java/nio/channels/DatagramChannel/PromiscuousIPv6.java

```diff
@@ -194,19 +201,22 @@ static void test(ProtocolFamily family,
         }
     }
 
+    private static boolean supportedByPlatform() {
```

Comment:
This method too, I think it needs a comment to this method to say that it returns true if platform allows an IPv6 socket join an IPv4 multicast group without interference.